common.tf

```diff
@@ -1,10 +1,11 @@
 locals {
-  az_count                         = local.environment == "prod" ? 2 : 1
-  ingest_raw_cache_bucket_name     = "${local.environment}-dr2-ingest-raw-cache"
-  ingest_staging_cache_bucket_name = "${local.environment}-dr2-ingest-staging-cache"
-  pre_ingest_step_function_name    = "${local.environment_title}-ingest-step-function"
-  additional_user_roles            = local.environment == "intg" ? [data.aws_ssm_parameter.dev_admin_role.value] : []
-  files_dynamo_table_name          = "${local.environment}-dr2-files"
+  az_count                                = local.environment == "prod" ? 2 : 1
+  ingest_raw_cache_bucket_name            = "${local.environment}-dr2-ingest-raw-cache"
+  ingest_staging_cache_bucket_name        = "${local.environment}-dr2-ingest-staging-cache"
+  pre_ingest_step_function_name           = "${local.environment_title}-ingest-step-function"
+  additional_user_roles                   = local.environment == "intg" ? [data.aws_ssm_parameter.dev_admin_role.value] : []
+  files_dynamo_table_name                 = "${local.environment}-dr2-files"
```

Comment:
Is `files` the best name for this table as it also contains folders and assets?

Reply:
https://en.wikipedia.org/wiki/Everything_is_a_file